extensions/amp-ad-network-fake-impl/0.1/data/fake_amp_ad_with_iframe_transport.html

```diff
@@ -1,20 +1,25 @@
 <!doctype html>
 <html amp4ads>
   <head>
+    <title>Ad with 3p Analytics and amp-ad-exit</title>
     <meta charset=utf-8>
     <script async src='https://cdn.ampproject.org/v0.js'></script>
-    <script async custom-element=amp-analytics src='https://cdn.ampproject.org/v0/amp-analytics-0.1.js'></script>
+    <script async custom-element='amp-analytics' src='https://cdn.ampproject.org/v0/amp-analytics-0.1.js'></script>
+    <script async custom-element='amp-ad-exit' src='https://cdn.ampproject.org/v0/amp-ad-exit-0.1.js'></script>
     <style amp4ads-boilerplate>body{visibility:hidden}</style>
     <meta name=viewport content=width=device-width,minimum-scale=1>
   </head>
   <body>
-    <a href="/landing.html">
-      <amp-img id=img src='https://upload.wikimedia.org/wikipedia/commons/6/6e/Golde33443.jpg' width=256 height=300></amp-img>
-      <p class=frob>
-        By Golden Trvs Gol twister (Own work) [CC BY-SA 4.0 (http://creativecommons.org/licenses/by-sa/4.0)], via Wikimedia Commons
-      </p>
-    </a>
-    <amp-analytics type="testVendor">
+    Before using this example ad, please add the following to
+    extensions/amp-analytics/0.1/vendors.js:
+    <pre>
+      'example-3p-vendor': {
```

Comment:
I am thinking we should add a fake analytics vendor in our vendor lists for testing purposes. Just like what we did with ad type `_ping_`. Let's talk about that when add integration test.

Reply:
Ok, that sounds fine.